src/tbp/monty/frameworks/models/motor_policies.py

```diff
@@ -288,10 +288,10 @@ def is_motor_only_step(self, state: MotorSystemState) -> bool:
         """
         agent_state = self.get_agent_state(state)
 
-        if "motor_only_step" in agent_state.keys() and agent_state["motor_only_step"]:
-            return True
-
-        return False
+        # FIXME: "motor_only_step" is not a valid AgentState key (based on type).
+        return bool(
+            "motor_only_step" in agent_state.keys() and agent_state["motor_only_step"]
```

Comment:
If `agent_state` is a dictionary (I don't know if it is here), then `keys()` isn't needed to check if the key is in the dictionary.
```suggestion
            "motor_only_step" in agent_state and agent_state["motor_only_step"]
```